WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -326,26 +326,30 @@ public async Task<InputRegistrationResponse> RegisterInputAsync(InputRegistratio
 
 		public async Task ReadyToSignAsync(ReadyToSignRequestRequest request, CancellationToken cancellationToken)
 		{
+			Alice? alice;
+
 			using (await AsyncLock.LockAsync(cancellationToken).ConfigureAwait(false))
 			{
-				if (Rounds.FirstOrDefault(r => r.Id == request.RoundId) is not Round round)
+				var round = Rounds.FirstOrDefault(r => r.Id == request.RoundId);
+				if (round is null)
```

Comment:
this diff doesn't actually do anything, it's only necessary for `alice`

Reply:
Fixed